### ISSUE TITLE
Support table

### ISSUE THREAD

**Issue by attila-lin** [robertknight/ocrs#39]
support table-ocr to csv will be nice

**Comment by robertknight** [comment]
I agree this would be a useful addition in the future. My focus at the moment is basic recognition accuracy and multi-language support. I imagine table detection/extraction would slot in somewhere at the layout analysis stage.
